docs/hub-deployment-guide/runbooks/phase3/initial-hub-setup.md

```diff
@@ -0,0 +1,176 @@
+(hub-deployment-guide:runbooks:phase3.1)=
+# Phase 3.1: Initial setup
+
+This phase is about configuring an initial, basic hub for the community to try it on. This hub will have basic customizations, like appearance, daskhub/basehub, a simple authentication mechanism and a list of initial admin users able to access it.
+
+## Definition of ready
+
+The following lists the information that needs to be available to the engineer before this phase can start.
+
+- Name of the hub
+- Dask gateway?
+- Splash image
```

Comment:
```suggestion
- Landing page image
```

Reply:
These are coming from the new hub issue https://github.com/2i2c-org/infrastructure/issues/3906. I will keep them on par with what's in the issue template and update it or remove it completely once we standardize the new format in an issue template. Does this sound ok to you?

Reply:
Absolutely!